webos-js/package.json

```diff
@@ -7,17 +7,22 @@
     "scripts": {
         "loc": "loctool -2 --xliffStyle custom --localeMap es-CO:es,fr-CA:fr --localeInherit en-AU:en-GB",
         "debug": "node --inspect-brk node_modules/loctool/loctool.js -2 --xliffStyle custom --localeMap es-CO:es --localeInherit en-AU:en-GB",
+        "clean": "rm -rf *.xliff resources",
         "loc-generate": "loctool generate -2 -x xliffs --projectType custom --sourceLocale en-KR --resourceFileTypes json=webos-json-resource --plugins webos-javascript,webos-appinfo-json  --projectId sample-webos-js -l de-DE,en-AU,en-US,en-GB,en-JP,es-ES,es-CO,fr-CA,fr-FR,ja-JP,ko-KR,ko-US --localeMap es-CO:es,fr-CA:fr --localeInherit en-AU:en-GB,en-JP:en-GB",
         "loc-generate-debug": "node --inspect-brk node_modules/loctool/loctool.js generate -2 -x xliffs --projectType custom --sourceLocale en-KR --resourceFileTypes json=webos-json-resource --plugins webos-javascript,webos-appinfo-json --projectId sample-webos-js -l de-DE,en-AU,en-US,en-GB,en-JP,es-ES,es-CO,fr-CA,fr-FR,ja-JP,ko-KR,ko-US --localeMap es-CO:es,fr-CA:fr --localeInherit en-AU:en-GB,en-JP,en-GB",
-        "clean": "rm -rf *.xliff resources",
-        "test": "npm run clean;npm run loc;node test/testResources.js; node test/testResourceLocalize.js",
-        "test-generate": "npm run clean;npm run loc-generate;node test/testResources.js; node test/testResourceGenerate.js"
+        "test": "node test/testResources.js; node test/testResourceLocalize.js",
+        "test-generate": "node test/testResources.js; node test/testResourceGenerate.js",
+        "test:all": "npm-run-all clean loc-generate test-generate",
```

Comment:
Should this be:

```
"test:all": "npm-run-all clean loc-generate test",
```

???

Reply:
ah, yes, it should be that. ^^; I fixed it. Thanks!